initials.c

```diff
@@ -0,0 +1,13 @@
+#include <stdio.h>
```

Comment:
Perfect - my only suggestion is to add a program description at the top (in a comment)